client/reducers/auth.js

```diff
@@ -4,15 +4,17 @@ import { SET_CURRENT_USER } from '../actions/types';
 
```

Comment:
file naming convention is a bit confusing, the action creator is called `handleLoginRequest` and the reducer is called `auth`